pkg/backends/graphite/graphite_test.go

```diff
@@ -135,7 +145,6 @@ func TestSendMetricsAsync(t *testing.T) {
 		}
 	}()
 	defer acceptWg.Wait()
-	defer l.Close()
```

Comment:
This is #161, this is:
- duplicate a line from above
- executing before `acceptWg.Wait()`, leading to the socket being closed before the goroutine has accepted from it

Reply:
I think the idea here was to close the socket and unblock the accept goroutine if something goes wrong in the main one. It should be ok to close multiple times, I think. Anyway, I'm not saying this is correct code and doesn't have bugs. Just trying to recollect why it is like that and I may be misremembering. Not opposed to making this change at all.

Reply:
Makes sense.  I can re-factor it if we start seeing failures again for sure, but this was close enough.